src/components/ContactInfo/ContactInfo.vue

```diff
@@ -283,6 +283,10 @@ export default class ContactInfo extends Vue {
   @Prop({ default: false })
   private invalidSection!: boolean
 
+  /** Option to disable phone requirement. */
+  @Prop({ default: false })
+  private optionalPhone!: boolean
```

Comment:
Props should be readonly (and no !).

Oops, please fix the other props, too.

Reply:
Copy that, will do!

Reply:
Updated

Reply:
Thanks.

Next time you see a prop with a `!`, it can be removed. (That's a signal to the compiler that the variable might initially be undefined, but that only applies to getters and actions, not props.) (This is not the "non-null assertion operator".)

Reply:
Ahh i see. 
Considering we also set defaults in a lot of instances. 

Will do 👍